src/tools/rados/rados.cc

```diff
@@ -94,6 +94,7 @@ void usage(ostream& out)
 "   setxattr <obj-name> attr val\n"
 "   rmxattr <obj-name> attr\n"
 "   stat <obj-name>                  stat the named object\n"
+"   stat2 <obj-name>                 stat2 the named object (with high precision time)\n"
```

Comment:
the prefix should be simply `rados` or `tools/rados`.

Reply:
for the commit message right? fixing

Reply:
yeah =)